pom.xml

```diff
@@ -171,6 +172,12 @@
                     <parallel>all</parallel>
                     <useUnlimitedThreads>true</useUnlimitedThreads>
                     <forkCount>1C</forkCount>
+                    <systemProperties>
+                        <property>
+                            <name>ui.loading.timeout</name>
+                            <value>${ui.loading.timeout}</value>
+                        </property>
+                    </systemProperties>
```

Comment:
[nit] `systemProperties` is deprecated in favor of `systemPropertyVariables` in recent versions of Maven. See [here](https://maven.apache.org/surefire/maven-surefire-plugin/examples/system-properties.html) for an example.